webpack.config.js

```diff
@@ -101,7 +101,8 @@ const config = {
   },
 
   plugins: [
-    new webpack.optimize.DedupePlugin(),
+    // faster without it and now we don't need this
```

Comment:
sure, so it can be moved into production plugins section or we should have different configuration for prod and dev ( or one more for testing ) , similar to angular class webpack starter config, what do you think ?